Exercises/strings/strings.js

```diff
@@ -0,0 +1,20 @@
+// STRINGS
+// 1. Assign string with unicode to a variable.
+let unicodeString = 'Example of unicode symbol is \u00B9';
+console.log(unicodeString);
+
+// 2. Declare a variable with string with escaped symbols.
+let escapeSymbol = '\u00A5';
```

Comment:
it is not escaped, but unicode symbol

Reply:
Changed to:

const escapeSymbols = 'Hello,\nthis is text on new line\t and this is text with tabulation';
console.log(escapeSymbols);